src/styles/components/newLocationForm.scss

```diff
@@ -0,0 +1,3 @@
+.form-container {
```

Comment:
.form-container probably repeats in multiple places. Should you pull up?

Reply:
I changed the class name to 'new-location-form-container' since each form has a different styling

Reply:
Why does each form have different styling? form styling should be consistent throughouth the app